web/src/lib/elements/ui/dataTable/data-table.svelte

```diff
@@ -184,10 +242,24 @@
       {Object.keys($selectedDataIds).length} von{' '}
       {$rows.length} Zeile(n) ausgewählt.
     </div>
-    <Button disabled={!$hasPreviousPage} on:click={() => ($pageIndex = $pageIndex - 1)} size="sm" variant="outline"
+    <Button
+      disabled={!$hasPreviousPage}
+      on:click={async () => {
+        $pageIndex = $pageIndex - 1;
+        await getData();
```

Comment:
Since you'll need to get the data whenever the pageIndex changes you should consider using something like `$: $pageIndex, await getData()`